### PR TITLE
🌱  Update for CAPM3 and IPAM release-1.7 and BMO 0.6.0

### PR DESCRIPTION
**Pull request by adilGhaffarDev**
Updating prow config to accommodate CAPM3 and IPAM release-1.7 and BMO release-0.6.0

### CODE REVIEW

File: jenkins/README.md [hunk 1]
```diff
@@ -49,6 +49,10 @@ clean up and deletion operations, there are separate triggers phrases as below:
   API version v1beta1 and branch main on Ubuntu
 * **/keep-test-centos-integration-main** run keep integration tests with CAPM3
   API version v1beta1 and branch main on CentOS
+* **/keep-test-ubuntu-integration-release-1-7** run keep integration tests with
+  CAPM3 API version v1beta1 and branch release-1.7 on Ubuntu
+* **/keep-test-centos-integration-release-1-7** run keep integration tests with
+  CAPM3 API version v1beta1 and branch release-1.7 on CentOS
```
Comment (kashifest) [inline jenkins/README.md:55]: arent we removing keep tests ?

Comment (adilGhaffarDev) [reply jenkins/README.md]: I have rebased now they are gone.

**Review by smoshiur1237 — COMMENTED** [review]
/lgtm

**Review by kashifest — COMMENTED** [review]
/approve
cc @mboukhalfa @tuminoid

**Review by Sunnatillo — COMMENTED** [review]
/lgtm

**Comment by adilGhaffarDev** [comment]
last failure was flake

**Pull request merged**